node/process.ts

```diff
@@ -220,6 +221,26 @@ class Process extends EventEmitter {
     return this;
   }
 
+  /**
+   * Returns the current high-resolution real time in a [seconds, nanoseconds]
+   * tuple.
+   *
+   * `time` is an optional parameter that must be the result of a previous process.hrtime() call to diff with the current time.
+   *
+   * These times are relative to an arbitrary time in the past, and not related to the time of day and therefore not subject to clock drift. The primary use is for measuring performance between intervals.
+   * https://nodejs.org/api/process.html#process_process_hrtime_time
```

Comment:
Probably worth mentioning that resolution is `1ms` if `--allow-hrtime` is not present